src/components/Sidebar/NavContent.jsx

```diff
@@ -250,25 +250,24 @@ function NavContent({ handleDrawerToggle }) {
                       <Trans>Stake</Trans>
                     </Typography>
                   </Link>
-
-                  {/* NOTE (appleseed-olyzaps): OlyZaps disabled until v2 contracts */}
-                  {/* <Link
-                    component={NavLink}
-                    id="zap-nav"
-                    to="/zap"
-                    isActive={(match, location) => {
-                      return checkPage(match, location, "zap");
-                    }}
-                    className={`button-dapp-menu ${isActive ? "active" : ""}`}
-                    onClick={handleDrawerToggle}
-                  >
-                    <Box display="flex" alignItems="center">
-                      <SvgIcon component={ZapIcon} color="primary" />
-                      <Typography variant="h6">OlyZaps</Typography>
-                    </Box>
-                  </Link> */}
-
-                  {EnvHelper.isGiveEnabled(location.search) ? (
+                  {isFeatureEnabled(OLYZAPS_ENABLED) && (
+                    <Link
+                      component={NavLink}
+                      id="zap-nav"
+                      to="/zap"
+                      isActive={(match, location) => {
+                        return checkPage(match, location, "zap");
+                      }}
+                      className={`button-dapp-menu ${isActive ? "active" : ""}`}
+                      onClick={handleDrawerToggle}
+                    >
+                      <Box display="flex" alignItems="center">
+                        <SvgIcon component={ZapIcon} color="primary" />
+                        <Typography variant="h6">OlyZaps</Typography>
+                      </Box>
+                    </Link>
+                  )}
+                  {isFeatureEnabled(GIVE_ENABLED) && (
```

Comment:
previously we had GIVE turned on by default (even if the feature flag was blank).